e2e_test/udf/sql_udf.slt

```diff
@@ -1,325 +1,346 @@
 statement ok
 SET RW_IMPLICIT_FLUSH TO true;
 
+#############################################
+# Basic tests for anonymous / named sql udf #
```

Comment:
I think anonymous function means function without a name. usually a closure. Better to just call them functions with named parameters.

Reply:
I think unnamed / named sql udf is a good choice then. 😄

Reply:
"unnamed" is still "anonymous"'s synonyms.. Please try to avoid making up terms that may lead to confusion. 

<img width="660" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/37948597/ac161966-1643-4dca-b78b-40e181f523dd">

Reply:
Then let's just stick with "sql udf with [named / unnamed] parameters", which should be precise and clear in most contexts.